SPECS/jurand/jurand.spec

```
@@ -0,0 +1,86 @@
+Name:           jurand
+Version:        1.3.2
+Release:        4%{?dist}
+Summary:        A tool for manipulating Java symbols
+License:        Apache-2.0
+Vendor:         Microsoft Corporation
+Distribution:   Azure Linux
+URL:            https://github.com/fedora-java/jurand
+
+Source0:        https://github.com/fedora-java/jurand/archive/refs/tags/%{version}.tar.gz#/%{name}-%{version}.tar.gz
+
+BuildRequires:  gcc-c++
+
+BuildRequires:  diffutils
+BuildRequires:  make
+BuildRequires:  rubygem-asciidoctor
+
+Obsoletes:      javapackages-extra < 6.2.0
+
+%description
+The tool can be used for patching .java sources in cases where using sed is
+insufficient due to Java language syntax. The tool follows Java language rules
+rather than applying simple regular expressions on the source code.
+
+%prep
+%setup -q
+
+%build
+%{make_build} test-compile manpages
+
+%install
+export buildroot=%{buildroot}
+export bindir=%{_bindir}
+export rpmmacrodir=%{_rpmmacrodir}
+export mandir=%{_mandir}/man7
+
+./install.sh
+
+%check
+make test
+
+%files -f target/installed_files
+%dir %{_rpmconfigdir}
+%dir %{_rpmmacrodir}
+%license LICENSE NOTICE
+%doc README.adoc
+
+%changelog
+* Thu Mar 21 2024 Riken Maharjan <rmaharjan@microsoft.com> - 1.3.2-4
+- Initial CBL-Mariner import from Fedora 40 (license: MIT).
```

Need to indicate that we verified the license
```suggestion
- Initial CBL-Mariner import from Fedora 40 (license: MIT).
- License Verified
```